jp/components/drop_down.md

```diff
@@ -0,0 +1,283 @@
+    <button igxButton="raised" (click)="toggleDropDown()">Countries</button>
+    <igx-drop-down>
+        <igx-drop-down-item *ngFor="let item of items" isDisabled={{item.disabled}} isHeader={{item.header}}>
+            {{ item.field }}
+        </igx-drop-down-item>
+    </igx-drop-down>
+</div>
+
+```
+
+```typescript
+// dropdown.component.ts
+
+...
+    @ViewChild(IgxDropDownComponent) public igxDropDown: IgxDropDownComponent;
+
+    public items: any[] = [
+        { field: "EU", header: true },
+        { field: "Germany" },
+        { field: "Bulgaria" },
+        { field: "UK", disabled: true }];
+
+    constructor() {
+    }
+
+    public ngOnInit() {
+    }
+
+    public toggleDropDown() {
+        this.igxDropDown.toggle();
+    }
+
+```
+
+サンプルが正しく構成されると、EU ヘッダーの下に国の一覧がグループ形式で表示され、UK は非インタラクティブな項目として表示されます。
+
+<div class="sample-container" style="height: 280px">
+    <iframe id="dropdown-sample-3-iframe" src='{environment:demosBaseUrl}/dropdown-sample-3' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="dropdown-sample-3-iframe" data-demos-base-url="{environment:demosBaseUrl}">                stackblitz で開く
+    </button>
+</div>
+
+<div class="divider--half"></div>
+
+### API まとめ
+以下の表は、便利な **igx-drop-down**  コンポーネントの入力、出力、およびメソッドを示します。
+
+#### 入力
+以下の入力は **igx-drop-down** コンポーネントで利用できます。
+
+| 名前 | タイプ | 説明 |
+| :--- | :--- | :--- |
+| `width` | 文字列 | コントロールのタブ幅を設定します。 |
+| `height` | 文字列 | コントロールのタブの高さを設定します。 |
+| `allowItemsFocus` | ブール値 | フォーカスを取得する項目を許可します。 |
+
+<div class="divider--half"></div>
+
+#### 出力
+以下の出力は **igx-drop-down** コンポーネントで利用できます。
+
+| 名前 | キャンセル可能 | 説明 | パラメーター
+| :--- | :--- | :--- | :--- |
+| `onSelection` | false | 選択を完了する前にアイテム選択が変更されるときに発生されます。| `{ oldSelection: string, newSelection: string, event?: Event }` |
+| `onOpening` | false | ドロップダウンが開かれる前に発生されます。 |
+| `onOpened` | false | ドロップダウンが開いているときに発生します。 |
+| `onClosed` | false | ドロップダウンが閉じているときに発生します。 |
+
+#### メソッド
+以下のメソッドは **igx-drop-down** コンポーネントで利用できます。
+
+| シグネチャ | 説明 |
+| :--- | :--- | :--- |
+| `toggle()` | ドロップダウンの開く/閉じるを切り替えます。 |
+| `setSelectedItem(index: number)` |ドロップダウン項目をインデックスで選択します。 |
+| `open()` | ドロップダウンを開きます。|
+| `close()` | ドロップダウンを閉じます。 |
+
+#### ゲッター
+以下のゲッターは **igx-drop-down** コンポーネントで利用できます。
+
+| 名前 | タイプ | 説明 |
+| :--- | :--- | :--- |
+| `selectedItem` | `number` | 選択項目を取得します。|
+| `items` | `QueryList of IgxDropDownItemComponent` | Gヘッダー以外のすべての項目を取得します。 |
+| `headers` | `IgxDropDownItemComponent[]` | ヘッダー項目を取得します。 |
+
+
+以下の表は、便利な **igx-drop-down-item** コンポーネントの入力、出力、およびメソッドを示します。
+
+#### 入力
+以下の入力は **igx-drop-down-item** コンポーネントで利用できます。
+
+| 名前 | タイプ | 説明 |
+| :--- | :--- | :--- |
+| `isHeader` | ブール値| 項目がグループ ヘッダーかどうかを定義します。 |
+| `isDisabled` | ブール値| 指定した項目を無効にします。 |
+| `isFocused` | ブール値| 指定した項目がフォーカスされるかどうかを定義します。|
+
+#### ゲッター
+以下のゲッターは **igx-drop-down** コンポーネントで利用できます。
```

Comment:
This should be `**igx-drop-down-item**`.

Reply:
Added.